hacspec-scrambledb/scrambledb/src/lib.rs

```diff
@@ -208,6 +260,6 @@ pub mod finalize;
 
 pub mod error;
 
-#[cfg(feature = "wasm")]
-pub mod wasm_demo;
+//#[cfg(feature = "wasm")]
```

Comment:
Why comment this out?

Reply:
I had commented this out because I hadn't updated the demo to the new double encryption method. I've done so now.